### DISCUSSION TITLE
Adding support for Message type

### DISCUSSION THREAD

**Discussion by mycroftcanner** [sbertix/Swiftagram#225] [Ideas]
Hi,

It seems the type `Conversation` just uses the `[Wrapper]` type for the `messages` property.

Shall I implement a concrete `Message` type that conforms to the `Wrapped` protocol? 

Thanks

**Comment by sbertix** [comment]
Hey @mycroftcanner,

I never added it cause adding specific models for all message types never fit with the time constraints I have to maintain the project. 

If you'd like to work on this and push a PR it'd be more than welcome 😊

**Reply by mycroftcanner** [reply]
No problem, I am working on a bunch of changes. I will push them once they are tested. In the meantime, I'll keep making posts here to confirm I'm going about it the right way.


Thanks again

**Reply by sbertix** [reply]
Great 💪 thank you. 

Make sure you create different PRs for unrelated features as to speed up release.

**Reply by mycroftcanner** [reply]
Will do! Thanks
